linera-views/src/scylla_db.rs

```diff
@@ -148,6 +152,48 @@ impl ScyllaDbClient {
         })
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 143]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Shouldn't this key size check also be done on `read_multi_values_internal`?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/2196?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
That check is present with
```
        let num_keys = keys.len();
        if num_keys == 0 {
            return Ok(Vec::new());
        }
```

Reply:
That's checking if we're passing 0 keys in, not the size of the keys themselves, like this checks. Unless I'm missing something

Reply:
Yes, you are correct.
Corrected.